packages/components/src/index.ts

```diff
@@ -1,4 +1,5 @@
 export { default as Button } from './button';
+export type { ButtonProps } from './button';
```

Comment:
I renamed this file from `.js` to `.ts` so that we can export this type. That said, is it "best practice" to export types like this?

Reply:
The syntax looks correct to me :)